Berksfile

```diff
@@ -3,8 +3,5 @@ source 'https://supermarket.chef.io'
 metadata
 
 group :integration do
-  cookbook 'sudo' # Use '< 5.0.0' with Chef < 13
```

Comment:
We need this for compatibility with older Chefs.

Reply:
Please see https://github.com/DataDog/chef-datadog/pull/820/files#r839644281